fashrevwall/twitter_client/TwitterClient.py

```diff
@@ -4,11 +4,13 @@
 """
 import os
 import tweepy
+import psycopg2
```

Comment:
We shouldn't use psycopg2 to access a postgres DB but the django way to access the elements in the model. To insert the data, we just create a Tweet model and save it, like in lines 83 and 86. We don't need to say which database or table, or use any SQL. The Django framework already provides these mechanism in a blackbox way, and understand what database is and what's the mechanism to do it by looking at the setting.py.